bentoml/_internal/models/model.py

```diff
@@ -163,25 +158,6 @@ def path(self) -> str:
     def path_of(self, item: str) -> str:
         return self._fs.getsyspath(item)
 
-    def save(
```

Comment:
Isn't this the same with line 78?

Reply:
No, it's OSFS-specific and runs `shutil.move`.

Reply:
I saw some similar code in `bento.py`, should we remove that as well?

Reply:
and could you fix the CI/codestyle_check?

Reply:
I thought I'd only do this one, since technically it's nice that the files don't get copied and we aren't doing save for bentos anywhere else, but I can.